### PR TITLE
Update appc/spec version to 0.8.1

### PR DESCRIPTION
**Pull request by tmrts**
Updates appc/spec dependency to `0.8.1`

### CODE REVIEW

**Comment by tmrts** [comment]
cc @yifan-gu

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
 	"GoVersion": "go1.6",
-	"GodepVersion": "v63",
+	"GodepVersion": "v67",
```
Comment (yifan-gu) [inline Godeps/Godeps.json]: Revert the change?

**Comment by tmrts** [comment]
Closing in favor of #25656
